lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -197,14 +197,21 @@ def private_marketing_app_error(manifest)
 
         def oauth_error(manifest)
           return unless manifest.oauth
-
+          oauth_errors = []
           missing = OAUTH_REQUIRED_FIELDS.select do |key|
             manifest.oauth[key].nil? || manifest.oauth[key].empty?
           end
 
           if missing.any?
-            ValidationError.new('oauth_keys.missing', missing_keys: missing.join(', '), count: missing.length)
+            oauth_errors << \
+              ValidationError.new('oauth_keys.missing', missing_keys: missing.join(', '), count: missing.length)
+          end
+
+          unless manifest.parameters.any? { |param| param.type == 'oauth' }
+            oauth_errors << ValidationError.new('oauth_parameter_required',
+                                                link: 'https://developer.zendesk.com/apps/docs/developer-guide/manifest#oauth')
```

Comment:
Idea: Should we move the link to a constant, in case we need to change it?

Reply:
done